semantic_conventions/scope/exporter/exporter.yaml

```
@@ -0,0 +1,30 @@
+groups:
+  - id: otel.scope
+    prefix: otel.scope
+    type: resource
+    brief: Span attributes used by non OTLP exporters to represent OpenTelemetry Scope's concepts.
```

```suggestion
    brief: Attributes used by non OTLP exporters to represent OpenTelemetry Scope's concepts.
```
since the scope also applies to metrics and logs, right?